packages/aserto/src/aserto/client/directory/aio/__init__.py

```diff
@@ -0,0 +1,469 @@
+from typing import Dict, List, Optional, Tuple, TypedDict
+
+import grpc.aio as grpc
+from aserto.directory.common.v2 import (
+    Object,
+    ObjectIdentifier,
+    ObjectTypeIdentifier,
+    PaginationRequest,
+    PaginationResponse,
+    PermissionIdentifier,
+    Relation,
+    RelationIdentifier,
+    RelationTypeIdentifier,
+)
+from aserto.directory.exporter.v2 import ExporterStub
+from aserto.directory.importer.v2 import ImporterStub
+from aserto.directory.reader.v2 import (
+    CheckPermissionRequest,
+    CheckRelationRequest,
+    GetObjectManyRequest,
+    GetObjectRequest,
+    GetObjectsRequest,
+    GetObjectsResponse,
+    GetRelationRequest,
+    GetRelationsRequest,
+    GetRelationsResponse,
+    ReaderStub,
+)
+from aserto.directory.writer.v2 import (
+    DeleteObjectRequest,
+    DeleteRelationRequest,
+    SetObjectRequest,
+    SetRelationRequest,
+    WriterStub,
+)
+from grpc import ChannelCredentials, StatusCode, ssl_channel_credentials
+
+
+class NotFoundError(Exception):
+    pass
+
+
+class Directory:
+    def __init__(self, channel: grpc.Channel, api_key: str, tenant_id: str) -> None:
+        self._channel = channel
+        self._metadata = self._get_metadata(api_key=api_key, tenant_id=tenant_id)
+        self.reader = ReaderStub(self._channel)
+        self.writer = WriterStub(self._channel)
+        self.importer = ImporterStub(self._channel)
+        self.exporter = ExporterStub(self._channel)
+
+    @classmethod
+    async def connect(cls, *, address: str, api_key: str, tenant_id: str, ca_cert: str):
+        channel = grpc.secure_channel(
```

Comment:
Shouldn't this be `channel = await grpc.secure_chanel(...`?
How are you testing this?